sbpy/activity/gas/tests/test_prodrate_remote.py

```diff
@@ -10,8 +10,8 @@
 from astroquery.jplspec import JPLSpec
```

Comment:
Could you add a header to identify the license?

 `# Licensed under a 3-clause BSD style license - see LICENSE.rst`